pkg/controlplane/control/manager.go

```diff
@@ -193,10 +233,87 @@ func (m *Manager) AddImport(ctx context.Context, imp *v1alpha1.Import) (err erro
 	return m.addImportService(ctx, imp, userService)
 }
 
+// deleteClDataplane deletes pod to ipToPod list.
+func (m *Manager) deleteEndpoint(esID types.NamespacedName) {
```

Comment:
I would rename `esID` to either `name`, `es`, `endpoint` or `endpointslice`.

Reply:
Also, would rename function to `deleteEndpointSlice` to be more accurate.
Same for `addEndpoint`.